app/views/admin/holdings.html.erb

```diff
@@ -25,7 +25,7 @@
           </button>
         </td>
         <td class='col-xs-2'><%= item.home_location %></td>
-        <td class='col-xs-2'></td>
+        <td class='col-xs-2'><%= SymphonyCurrLocRequest.new(barcode: item.barcode).current_location %></td>
```

Comment:
Working on a feature;  if you want to deploy as is for the demo, that's cool.   It's actually deployed to requests-stage right now.